src/reynir_correct/spelling.py

```diff
@@ -618,13 +642,13 @@ def stupid_backoff(w: str) -> float:
             )
         if log_prob > self._UNIGRAM_ACCEPT_THRESHOLD:
             # print(f"The original word {word} is above the threshold, returning it")
-            return word
+            return []
         # Otherwise, generate replacement candidates
         candidates = list(gen_candidates(original_word, word))
         if not candidates:
             # No candidates beside the word itself: return it
             # print(f"Candidate {word} is only candidate, returning it")
-            return word
+            return []
```

Comment:
Update the above comment accordingly, the word is not returned

Reply:
Done